### ISSUE TITLE
problem with mask_token

### ISSUE THREAD

**Issue by v-zixin** [neulab/awesome-align#32]
I'm trying to fine-tune on parallel data, and I get an error in the mask_token function in run_train.py

The error message is: Mask tensor can only take 0 and 1 values. (Line 146).

Could you offer some advice on how to fix this?

**Comment by zdou0830** [comment]
Hi, what's your pytorch and python version?

**Comment by zixinzeng-jennifer** [comment]
Thanks for the quick reply. I'm using Python 3.8.6, torch 1.9.0+cpu

**Comment by zdou0830** [comment]
Hi, it seems that I cannot reproduce this error on my end. Could you print out the detailed error message? 

Also, could you see if changing this line (https://github.com/neulab/awesome-align/blob/master/awesome_align/run_train.py#L154) to `mask_type = torch.bool` and running `python setup.py install` again would solve the issue?
